mysql-test/suite/versioning/t/vtmd.test

```diff
@@ -36,7 +36,7 @@ create or replace procedure check_vtmd (in vtmd_name varchar(64))
 begin
   set @tmp= concat('
     create or replace temporary table
-    tmp_vtmd as
+    tmp_vtmd with system versioning as
```

Comment:
Extension is: specifying just `WITH SYSTEM VERSIONING` for a table, i.e. implicit `GENERATED ALWAYS` and `PERIOD FOR`. This extension was specified in HDL but nothing was said about `CREATE TABLE FROM SELECT` case. And it looks like a separate feature.